packages/components/src/List/Toolbar/Toolbar.scss

```diff
@@ -27,11 +29,63 @@ $tc-list-toolbar-background-color: $concrete !default;
 		}
 	}
 
-	:global(.navbar-nav) {
-		a[role="menuitem"],
+	:global(.nav) {
+		a[role='menuitem'],
 		a:global(.dropdown-toggle) {
 			color: $black;
 			text-transform: capitalize;
+			font-weight: normal;
+		}
+	}
+
+	:global(.tc-actionbar-container) {
+		background-color: $tc-list-toolbar-background-color;
+		padding-right: 0;
+		height: $tc-list-toolbar-height;
+
+		:global(.navbar-left),
+		:global(.navbar-right) {
+			> * {
+				margin-right: $padding-normal;
```

Comment:
The last margin interfere with defining proper space round the bar.
What we need is to set space between elements, and not at the edge. Can you remove the margin-right of the last item please ?